cli/admin/src/main/java/io/pravega/cli/admin/utils/CLIConfig.java

```diff
@@ -40,7 +40,7 @@
     private static final Property<String> PASSWORD = Property.named("credentials.pwd", "");
     private static final Property<Boolean> TLS_ENABLED = Property.named("channel.tls", false);
     private static final Property<String> TRUSTSTORE_JKS = Property.named("trustStore.location", "");
-    private static final Property<Integer> TRUSTSTORE_ACCESS_TOKEN_TTL_SECONDS = Property.named("trustStore.access.token.ttl.seconds", 10);
+    private static final Property<Integer> TRUSTSTORE_ACCESS_TOKEN_TTL = Property.named("trustStore.access.token.ttl", 10);
```

Comment:
The **field** name is fine (no time unit). The property name (in the string) must have unit (`.seconds`).